README.md

```diff
@@ -1,9 +1,29 @@
 # TSE-Newsletter
 
-Please do not directly edit version0.html, this is the file you should reference! The newsletter.html file is the one we will use.
+[![GitHub last commit](https://img.shields.io/github/last-commit/TritonSE/TSE-Newsletter)](https://github.com/TritonSE/TSE-Newsletter/commits/main)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/TritonSE/TSE-Newsletter/pulls)
```

Comment:
Is there a way to change the colors so that they have the same shade of green? Or is there a reason for the different shades?

Reply:
Good catch! Fixed.